src/analysis/retail/evoker/devastation/modules/AplCheck/index.tsx

```diff
@@ -184,20 +141,14 @@ const COMMON_TOP: Rule[] = [
     spell: SPELLS.ETERNITY_SURGE,
     condition: cnd.or(
       cnd.buffPresent(TALENTS.DRAGONRAGE_TALENT),
-      cnd.and(
-        cnd.buffMissing(SPELLS.BLAZING_SHARDS),
-        cnd.optionalRule(avoidIfDragonRageSoon(13000)),
```

Comment:
i see you also ran into the "`optionalRule` inside `and` makes the whole rule optional" issue :upside_down_face: should maybe just make optional-in-and an error.

Reply:
I actually had the opposite happen, `cnd.optionalRule(avoidIfDragonRageSoon(13000)),`  check didn't become optional 🤔

Reply:
well the `console` error called confusing behavior for a reason lol